package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "beemo create-config prettier && beemo eslint \"./packages/*/{src,test,storybook}/**/*.{js,jsx,ts,tsx}\"",
```

Comment:
I think you've just moved storybook config, but since you've moved directories around just want to verify that this line still works for components/stories?

Reply:
This should work. I only move `storybook-config` dir to `.storybook`. 
The `stories` are in the same path as before.
Only the file `storybook/storybookInfo.js` is merged into `config.js`.